lib/browserify-plugins/id-hasher/id-hash-transform.js

```diff
@@ -7,7 +7,10 @@ module.exports = function idHashTransform(idToHashMap) {
     return new Transform({
         objectMode: true,
         transform(obj, enc, next) {
-            idToHashMap[obj.id] = common.hasher(obj.source);
+            // include id and source in hash to avoid situation where
+            // multiple modules have the same source code but different
+            // depedendencies somewhere down the line.
+            idToHashMap[obj.id] = common.hasher(obj.id + obj.source);
```

Comment:
`path.relatrive(__dirname, obj.source.path)` or something like that?

Reply:
yea, relative to process.cwd()? The scenario where determinism breaks down is where you have multiple asset servers on different machines trying to produce the same bundle. process.cwd would always contain the machine unique part that we want to trim out right? (Having trouble reasoning it out in my head)